### PR TITLE
JBDS-3408: Make p2StatsUrl a property

### PR DESCRIPTION
**Pull request by mickaelistria**
In order to override it in devstudio

### CODE REVIEW

**Comment by nickboldt** [comment]
+1.

File: parent/pom.xml [hunk 1]
```diff
@@ -334,7 +334,7 @@
 								<update.site.version>${project.version} (${buildQualifier})</update.site.version>
 								<target.eclipse.version>${targetEclipseVersion}</target.eclipse.version>
 							</symbols>
-							<p2StatsUrl>http://download.jboss.org/jbosstools/usage/installs/${project.version}/${BUILD_ALIAS}/${JOB_NAME}/${buildQualifier}</p2StatsUrl>
+							<p2StatsUrl>${[2StatsUrl}</p2StatsUrl>
```
Comment (nickboldt) [inline parent/pom.xml]: There's an extra square bracket here.

Comment (mickaelistria) [inline parent/pom.xml]: Thanks for finding the typo. I did push another version of the commit to this PR.

**Comment by mickaelistria** [comment]
About the fallback value for JOB_NAME, I don't think this has to be fixed as part of this change, unless there is a blocker issue I'm not aware of. The current mechanism is also sensible to missing properties, and so far, I've noticed no complaint on this topic.

**Comment by nickboldt** [comment]
If you build locally w/o JOB_NAME defined you get errors in Eclipse when you install because Eclipse tries to load the usage URL, which contains dollar signs. 

So, you really need a fallback value here. 

Suggest when JOB_NAME is not set (eg., when "hudson" profile not enabled) that the path segment be "local" instead. Then the hudson profile can override it using JOB_NAME instead.

Or, we could drop the JOB_NAME segment entirely, per Denis' suggestion:

https://github.com/jbdevstudio/jbdevstudio-product/pull/305/files#r29773552

**Comment by mickaelistria** [comment]
The question is more whether it's really something we want to discuss and fix as part of Beta2 (which is meant to be staged in a few hours), or if it's something we can delay to Beta2 ?

**Comment by mickaelistria** [comment]
Merged on both branch and master.
We can discuss the JOB_NAME issue in another Jira and another PR.

**Pull request merged**